### PR TITLE
Feature/r5

### PR DESCRIPTION
**Pull request by jkim333**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,19 @@
...
+
+MongoDB is used as the database to allow for persistent storage of data. It is the responsibility of NodeJS and ExpressJS to communicate with MongoDB. The frontend React app should not talk to the database as it is not a secure practice. MongoDB will be hosted on MongoDB Atlas.
+
+Mongoose is an Object Data Modelling library for MongoDB, which allows developers to enforce schema for MongoDB at the NodeJS application layer and provides tools and features to make it easier to work with mongoDB. Mongoose will be used to communicate with the MongoDB database.
+
+In summary, MongoDB is responsible as a database, ExpressJS and NodeJS are responsible as a backend web server, and React is responsible as a frontend app to interact with the users.
+
```
Comment (inverseswirl) [inline README.md:54]: Looks good to me

File: README.md [hunk 2]
```diff
@@ -48,4 +48,28 @@ In summary, MongoDB is responsible as a database, ExpressJS and NodeJS are respo
...
+![(Tablet) Calendar page](./docs/(Tablet)_Calendar_page.png)
+![(Tablet) Dashboard page](./docs/(Tablet)_Dashboard_page.png)
+
+![(Desktop) Signup page](./docs/(Desktop)_Signup_page.png)
+![(Desktop) Login page](./docs/(Desktop)_Login_page.png)
+![(Desktop) Home page](./docs/(Desktop)_Home_page.png)
+![(Desktop) Choose exercise page](./docs/(Desktop)_Choose_exercise_page.png)
+![(Desktop) Log entry page](./docs/(Desktop)_Log_entry_page.png)
+![(Desktop) Calendar page](./docs/(Desktop)_Calendar_page.png)
+![(Desktop) Dashboard page](./docs/(Desktop)_Dashboard_page.png)
```
Comment (inverseswirl) [inline README.md:99]: looks good to me

**Review by jarroddalefolino — COMMENTED** [review]
ok looks good. One thing is that with `docs/application_architecture_diagram.drawio.png` mongoose is a library that lives inside the Node (Express) App. Where as on the diagram it is seperate

**Comment by jkim333** [comment]
Good point Jarrod. Thanks. I will update the diagram.

File: docs/application_architecture_diagram.drawio [hunk 3]
```diff
@@ -1 +1 @@
...
...vwl5AE2N5ZBEgX98VlvFFTkgT0ikPifZIWnvP2V0JemS02l4Kmi1veQxJz7HibY+Me44TBgP8q4BdAXgDqwAWgsUFZFfAlD2DBstlaxZD3lgoOU8ky5pgxNMUItnAqBB801w250nzqRldgAFMI5qY6D2L5VKH5QQVfgVssSyfbPs64BUtF+tI8iWN+aYGkUmPjATnshittiNIFHclL8W+ixdmDy8mIJVv2XBhPzyTTfT75nzsnn+73l3bV+OvbuHliSZrHbB+WbkrGRB8ncagnNg9cr5ZMgnTjEZqdoOSI7aUq0RP04QtUhxH+FIgEJizJBnxhIu9MwJ27EGAeC4Ff4TazMAPCPVxxoxLh/oEQsK2Buk4L4GvQIodLilnyyTTSWeHYWFvKglJmWHLmnyOpeOnOm0WB98VszjQ5P4F0bZB9B2WCiJnk20mIM+/4HiYZT3HpytFajrLsz0blgm17Y8o1hJoHkYQRV0CzULP9ayaxqLg7RSCHcpdC+a4pmBO0CGYbX+WYI4h2C1PF3ysFDmrhkOZUKVdSwIs9kwNo13CUAtBjgsxK1S7mR0AGj0u9lp+X0t0AxrPiyZpe6Z6WFxh7HapFzoz4p+ovEhbLNcQy3a7xCpb4cnFIoZYP4HiaaBLyjq7A2RwvjOF+kitxBTCeWet+FEIs/mJasP3GnQHxGQ78E22/c8i2zPIHvPoEbs98jy8n+K/yWhqMo3hSoOrlKu0bhCroTrtPJXTetIrWztW84pLhkf2UDemFYtjNWmeRidJ/uNni2116OF9lh7B8TMc0nioLkOKC+xXOSuylgppwjXiq+KwOmSALZO/1Fy/tB70PjUeb/W2vbErjRQjLjaFg9J+2NuWZZdAtXdvNTb/AMGQNyXnHnxR0pyvRQTHmwaysAB5/OiGuHFTNBOklgBeR/crMQEJleypeb/sSgr9hB+cpbLWDryw7wxqn2Z3wKtp02NBg3ZSvx62/BLith31MderT9D0W9Bm+MV8orvaskwtyF8Oh3ikWU6B9eprup712nocFG9QldNBofdXWNh9F+A5nPbuNZ873Xev2J/53olO78MXklLooOP4djoS+NMOlPIO9+872KEbNVuRd6QRvb/nOG/sOeF/1XNsu9Vk/LDfagdvbTPt7HPbWfVCXzleymhWX6aL5dUvEmTyBw==</diagram></mxfile>
```
Comment (inverseswirl) [inline docs/application_architecture_diagram.drawio:1]: Great! looks good  Ji.

**Pull request merged**